caikit_nlp/modules/text_generation/peft_tgis_remote.py

```diff
@@ -111,12 +118,24 @@ def load(cls, model_path: str, load_backend: BackendBase) -> "PeftPromptTuningTG
         # we convert make it valid json compatible dict (aka doesn't have non string keys)
         log.debug("Prompt ID: %s", prompt_cache_id)
         log.debug("TGIS model ID: %s", base_model_name)
+
+        # Get all the valid prompt artifact files so they can be loaded after
+        # the connection is established
+        prompt_artifacts = [
+            os.path.join(model_path, config.get(config_key))
+            for config_key in [
+                PeftPromptTuning._ENCODER_KEY.name,
+                PeftPromptTuning._DECODER_KEY.name,
+            ]
+            if config.get(config_key)
```

Comment:
q - I'm a bit confused how any these keys (ENCODER/DECODER) would be expected to end up in the module config without changes to `save`?

Reply:
just kidding I see the `base_module` save now